src/app/api/Classes/user.ts

```diff
@@ -0,0 +1,4 @@
+export class UserClass {
```

Comment:
export class User {      (without extra class in name)

- [ ] remove class from name